developer/src/kmlmc/source/util/util.ts

```diff
@@ -41,8 +41,8 @@ export function loadFromFilename(filename: string): LexicalModelSource {
   // NOTE: transpile module does a very simple TS to JS compilation.
   // It DOES NOT check for types!
   let compilationOutput = ts.transpile(sourceCode, {
-    // Our runtime should support ES6 with Node/CommonJS modules.
-    target: ts.ScriptTarget.ES2015,
+    // Our runtime only supports ES3 with Node/CommonJS modules on Android 5.0
```

Comment:
Was the `const` keyword the main notable thing?  

Also, would we leave this in place when we decide to shift our minimum requirements higher than Android 5.0?  If we think we might lift the JS target in the future, it might be wise to add related notes here.

Reply:
Yes, `const` was the main thing we saw. But obviously given we accept arbitrary TS code, other ES2015+isms could easily creep in...